### DISCUSSION TITLE
게시글 내에 이미지 추가시 No Image로 뜹니다.

### DISCUSSION THREAD

**Discussion by taehuuun** [pronist/hELLO#197] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/43982651/219662685-c4548f75-1afc-4283-b31d-37097eaa05ae.png)

위 이미지 처럼 게시글에 이미지를 추가하였을때 No Image로 뜨는거 같습니다.
버전 문제인지 확실 하지 않지만 '23.02.17 기준 4.1.2 버전 사용중입니다.

 좀 기다려서 (약 3분 ~ 5분) 다시 확인 해보니 정상적으로 나오네요 업로드 처리되는 부분이 랜덤인듯 한데 스킨 문제는 아닐꺼라고 생각이 듭니다.

**Comment by pronist** [comment, marked as answer]
@taehuuun 안녕하세요.

이것은 저도 처음보는군요. 오늘 패치했나? 카카오에서 이미지 업로드에 큐를 사용하기라도 한 건지 기본 이미지까지 설정한 것을 보면 뭔가 건드리긴 했나봅니다. 사실 티스토리 쪽에서 잠수함 패치를 해버리면 스킨 개발자 입장에서는 뭐가 바뀐건지 알아차리기가 참 쉽지 않죠.
